### PR TITLE
feat(examination-timetable.js): change logic to show make submission button

### PR DESCRIPTION
**Pull request by Philip-Philippou1**
Change to logic for make a submission button.  We no longer want the +1 day logic.  

The button now shows if the deadline is today, we treat it as an end of day deadline now.

### CODE REVIEW

File: packages/forms-web-app/src/controllers/projects/examination-timetable.js [hunk 1]
```diff
@@ -31,13 +31,12 @@ const eventStates = [
...
 const eventSubmitButtonActive = (timetable) => {
-	const tomorrow = new Date();
-	tomorrow.setDate(tomorrow.getDate() + 1);
+	const currentDate = new Date();
 
 	return (
 		timetable.typeOfEvent === 'Deadline' &&
-		tomorrow < new Date(timetable.dateOfEvent) &&
-		(new Date(timetable.dateTimeDeadlineStart) < getDate() ||
+		currentDate < new Date(timetable.dateOfEvent).setHours(24, 0, 0, 0) &&
```
Comment (jonathangoulding) [inline packages/forms-web-app/src/controllers/projects/examination-timetable.js:38]: can this be a func  - 
`const dateIn24Hours = (date) => new Date(date).setHours(23, 0, 0, 0);`

occurs twice so would be nice to lift.

Comment (jonathangoulding) [reply packages/forms-web-app/src/controllers/projects/examination-timetable.js:38]: ### or something better named. 

and used in the two places instead.

Comment (Philip-Philippou1) [reply packages/forms-web-app/src/controllers/projects/examination-timetable.js]: Done - added setTimeToEndOfDay.

**Review by jonathangoulding — DISMISSED** [review]

File: packages/forms-web-app/src/controllers/projects/examination-timetable.js [hunk 2]
```diff
@@ -30,14 +30,15 @@ const eventStates = [
...
 const eventSubmitButtonActive = (timetable) => {
-	const tomorrow = new Date();
-	tomorrow.setDate(tomorrow.getDate() + 1);
+	const currentDate = new Date();
 
 	return (
 		timetable.typeOfEvent === 'Deadline' &&
-		tomorrow < new Date(timetable.dateOfEvent) &&
-		(new Date(timetable.dateTimeDeadlineStart) < getDate() ||
+		currentDate < setTimeToEndOfDay(new Date(timetable.dateOfEvent)) &&
```
Comment (jonathangoulding) [inline packages/forms-web-app/src/controllers/projects/examination-timetable.js:40]: new date not required as the function does new date

**Review by jonathangoulding — COMMENTED** [review]
Quick one

File: packages/forms-web-app/src/controllers/projects/examination-timetable.js (hunk 2 above)
Comment (Philip-Philippou1) [reply packages/forms-web-app/src/controllers/projects/examination-timetable.js]: Yes done.

**Review by jonathangoulding — APPROVED** [review]

**Pull request merged**